hw/dv/tools/dvsim/vcs.hjson

```diff
@@ -288,7 +297,9 @@
     {
       name: vcs_xprop
       is_sim_mode: 1
-      build_opts: ["-xprop={dv_root}/tools/vcs/xprop.cfg"]
+      build_opts: ["-xprop={dv_root}/tools/vcs/xprop.cfg",
+                   // Enable xmerge mode specific performance optimization
```

Comment:
I do not really understand this comment :)
-> Enable xmerge mode to optimize performance

Reply:
never mind, sorry my english stucked..